src/main/java/org/jenkinsci/test/acceptance/FallbackConfig.java

```diff
@@ -123,6 +126,14 @@ private WebDriver createWebDriver(TestName testName) throws IOException {
             capabilities.setCapability(LANGUAGE_SELECTOR, "en");
             capabilities.setCapability(LANGUAGE_SELECTOR_PHANTOMJS, "en");
             return new PhantomJSDriver(capabilities);
+        case "remote-webdriver-firefox":
+            String u = System.getenv("REMOTE_WEBDRIVER_URL");
+            if (StringUtils.isBlank(u)) {
+                throw new Error("remote-webdriver-firefox requires REMOTE_WEBDRIVER_URL to be set");
+            }
+            return new RemoteWebDriver(
+                    new URL(u), //http://192.168.99.100:4444/wd/hub
+                    DesiredCapabilities.firefox());
```

Comment:
Well since the `BROWSER` var is locally used for firefox or chrome it would be easy to add `remote-webdriver-chrome` in the future. I don't see a particular reason to add it here now?